Solutions/CiscoUmbrella/Analytic Rules/CiscoUmbrellaConnectionNon-CorporatePrivateNetwork.yaml

```diff
@@ -25,6 +25,8 @@ query: |
```

Comment:
1. Consider using datetime columns directly instead of the long data type. KQL works better with datetime than long. You can use update policies to convert unix time to the datetime data type during ingestion. Function causing the suggestion: unixtime_milliseconds_todatetime
2. Use the '==' operator instead of '=~' for case-sensitive comparisons.

**Disclaimer:** These suggestions are offered to enhance query efficiency and adherence to best practices. It is recommended to consider applying them for improved code quality, but their application is optional and context-dependent.